### PR TITLE
Added two new interfaces for DatabaseReader.cs and WebServiceClient.cs 

### PR DESCRIPTION
**Pull request by Nawaaz**
Dependency injection will be possible for those and still make use of the connectiontype and isp methods

### CODE REVIEW

**Comment by oschwald** [comment]
This makes sense. It looks like they need to be added to the VS project.

File: GeoIP2/GeoIP2.csproj [hunk 1]
```diff
@@ -64,7 +64,9 @@
     <Compile Include="Exceptions\HttpException.cs" />
     <Compile Include="Exceptions\InvalidRequestException.cs" />
     <Compile Include="Exceptions\OutOfQueriesException.cs" />
+    <Compile Include="IGeoIP2DatabaseReader.cs" />
```
Comment (Nawaaz) [inline GeoIP2/GeoIP2.csproj]: Added the missing files. :) Completely missed it in the first commit

**Comment by oschwald** [comment]
Thanks!

**Comment by Nawaaz** [comment]
My pleasure...

**Pull request merged**